lib/src/yaml_node.dart

```diff
@@ -36,6 +36,9 @@ abstract class YamlNode {
   /// [YamlList], it will return [this], since they already implement [Map] and
   /// [List], respectively.
   get value;
+
+  /// Returns a copy of this YAML node as a JSON-compatible type.
```

Comment:
If it is making a copy then the convention would be to name it `toJsonType`.

https://dart.dev/guides/language/effective-dart/design#prefer-naming-a-method-to___-if-it-copies-the-objects-state-to-a-new-object

We could also consider writing an adapter instead.